addons/l10n_vn_edi/data/neutralize.sql

```diff
@@ -0,0 +1,4 @@
+-- disable l10n_vn_edi integration
+UPDATE res_company
+   SET l10n_vn_edi_username = '',
+       l10n_vn_edi_password = '';
```

Comment:
Not sure but would it not be a good idea to set the value for test that you provide in the chatter of the task ? Or are these only for development probably ?

Reply:
Nope, we can't do that. In any case, the neutralize is here to avoid sending unwanted data during testing/... so it wouldn't be of much uses.